bin/API_Migration_scripts_readme/WebScrapper_Java_Code-OfficialCopy.ipynb

```diff
@@ -0,0 +1,783 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "361542ff",
+   "metadata": {},
+   "source": [
+    "# Python Scripts - Text extraction (webscraping) and matching contents of Java file(s)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a2f0458",
+   "metadata": {},
+   "source": [
+    "## Created for API Migration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ab88f421",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#imports\n",
+    "import os\n",
+    "import csv\n",
+    "import requests\n",
+    "from bs4 import BeautifulSoup\n",
```

Comment:
nit: in terms of a style guide, I don't like `from X import Y`. I prefer to use `import X as x` and then `x.Functionality`. This allows me to know where the various modules come from. In this case, since `bs4` is short enough, you can just use `bs4.BeautifulSoup`

Reply:
Will follow this approach in the future ma'am

Reply:
Is there a reason why it can't be fixed now?